website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -47,7 +47,7 @@ The fields that will be updated are:
 - Single sign-on URL &mdash; `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`
 - Audience URI (SP Entity ID) &mdash; `urn:auth0:<YOUR_AUTH0_ENTITYID>:{slug}`
 
-Sample steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud):
+Sample steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud and should coordinate making these changes with your identity provider admin):
```

Comment:
looks like important enough information so sugg removing the parenthesis. typically, parenthetical statements describe non-essential information

```suggestion
Below are sample steps to update. You must complete all of them to ensure uninterrupted access to dbt Cloud and you should coordinate with your identity provider admin when making these changes.
```